pom.xml

```diff
@@ -341,7 +341,7 @@
                         <configuration>
                             <!-- pointless to attempt incremental build if something like mvnw was changed
                                  (and also potentially wrong, given that independent-projects might not be built) -->
-                            <skipIfPathMatches>\.github[/\\].*|\.mvn[/\\].*|mvnw.*</skipIfPathMatches>
+                            <skipIfPathMatches>\.sdkmanrc|\.github[/\\].*|\.mvn[/\\].*|mvnw.*</skipIfPathMatches>
```

Comment:
That's the wrong property, you need to use `excludePathsMatching` instead.
`skipIfPathMatches` makes GIB disable itself if a changed file matches the pattern, effectively triggering a full build.

Reply:
Ah, okay